do.bat

```diff
@@ -0,0 +1 @@
+build.cmd
```

Comment:
Did you intend to commit these four `.bat` files?

Reply:
removed